Milestone 1: App Set-up and Structure

DuaneDave opened the pull request:
### In this milestone, we:

- Set up the app structure
- Add some stylings to app
- Add a function to add todo's

DuaneDave committed:
Set up todo structure with dummy texts

DuaneDave committed:
clear linter errors

DuaneDave committed:
Update readme file

DuaneDave committed:
clear linter error

DuaneDave committed:
clear linter error

DuaneDave committed:
clear linter error

DuaneDave committed:
clear linters

HAMMAS-SALEEM reviewed (commented):
# Invalid Pull Request :red_circle: :traffic_light: :rotating_light:

## Hey @DuaneDave :wave: 

**You have made a great effort on your project :blush: and I can see that but unfortunately I can not proceed with a code review of your pull request due to the following reasons :receipt:**

## 1. 
![lighthouse-missing](https://user-images.githubusercontent.com/93039359/183656920-887dc06b-1f61-4271-9d57-cb80acabf1fb.png)

- [ ] I can see that you have removed the **lighthouse** from your linters. Please add it and try to fix the linter errors manually. :+1: Linters are very important because they prevent you to make those errors that can break your application. :thinking:

### Once you implement all the changes you should be ready to submit your project for a review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

DuaneDave commented:
Hello @HAMMAS-SALEEM 👋 

I removed it because it kept giving me some issues that I cant fix and I don't know what to do so I removed it.
Pls can u help

HAMMAS-SALEEM commented:
Hey, @DuaneDave can you come on slack?

DuaneDave committed:
Update linters.yml

DuaneDave committed:
Remover server.js

DuaneDave committed:
Remove dist folder
